bspump/ipc/protocol.py

```diff
@@ -93,10 +96,13 @@ async def handle(self, source, stream, context):
 
 
 	def _line_codec_decoder(self, line_bytes):
-		line, _ = self.Codec.decode(
-			line_bytes
-		)
-		return line
+		try:
+			line, _ = self.Codec.decode(line_bytes)
+		except UnicodeDecodeError as err:
+			L.warning("Cannot decode line, replacing malformed data with backslash escape sequence: {}".format(err))
```

Comment:
Don't log warnings in the "main" processing pipeline, this will only overflow logs.